Week 3 content updates and transcript formatting revisions (weeks 1, 2, and 3)

jgagne opened the pull request:
This PR adds necessary info to week 3 front matter such as poster art, video duration and ID, related content links, etc. and adds formatting to transcripts (weeks 1, 2, and 3).

jgagne committed:
Add transcripts with basic formatting

- Add leading line break after front matter
- Educate quotes
- Replace double hyphen with em dash (with single space before and after)
- Replace “Take Five” with “Take 5”
- Spell out “OK” to “okay”
- Add separate paragraph for “Big Idea/Best Practice” when possible (when it’s a natural line break)
- Update title capitalization of “Big Idea/Best Practice Number”
- Use numerals for Big Idea/Best Practice” numbers

jgagne committed:
Format Big Idea/Best Practice lines

- Format Big Idea/Best Practice
- Add comma before “Number”
- Add separate paragraph for “Big Idea/Best Practice” when possible (when it’s a natural line break)

jgagne committed:
Update Week 3 Front Matter

Fill in the necessary info such as poster art, video duration and ID, related content links, etc.

jgagne committed:
Add emphasis, structure, and links

- Add emphasis for voice and tone
- Add better separation of “Big Idea/Best Practice” lines
- Add reference-style links

jgagne committed:
Add emphasis and links

- Add emphasis for voice and tone
- Add reference-style links

jgagne committed:
Add emphasis, structure, quotes, and links

- Add emphasis for voice and tone
- Add better separation of “Big Idea/Best Practice” lines
- Update format for conversational quotes
- Add reference-style links

jgagne committed:
Update Week 2 Transcripts (#226)

* Remove errant use of inline code

- Use <kbd> for keyboard commands
- Use <strong> for typed phrases

* Fix typo

Change 0 (numeral) to O (letter).

andrewpmiller commented on _data/take5s/GYM-5016.yml:
```diff
@@ -3,26 +3,28 @@ course_ID: GYM-5016
+#- label:
+#  url:
+#- label:
+#  url:
```
These can be removed (or commented out)

andrewpmiller commented on _data/take5s/GYM-5017.yml:
```diff
@@ -3,22 +3,26 @@ course_ID: GYM-5017
+#- label:
+#  url:
+#- label:
+#  url:
```
These can be removed (or commented out)

andrewpmiller commented on _data/take5s/GYM-5018.yml:
```diff
@@ -4,25 +4,25 @@ title: "Crafting Your Story"
+#- label:
+#  url:
+#- label:
+#  url:
```
These can be removed (or commented out)

andrewpmiller commented on _data/take5s/GYM-5019.yml:
```diff
@@ -4,21 +4,27 @@ title: "Self-Promotion Through Your Online Presence"
+#- label:
+#  url:
+#- label:
+#  url:
```
These can be removed (or commented out)

jgagne committed:
Add emphasis, structure, citations, and links

- Add emphasis for voice and tone
- Add better separation of scenarios
- Add <cite> for citing Webby award
- Add reference-style links

jgagne committed:
Add emphasis, structure, citations, and links

- Add emphasis for voice and tone
- Add better separation of “Big Idea/Best Practice” lines
- Add <cite> for citing social platforms
- Add reference-style links

jgagne committed:
Merge branch 'patch/update-week3-frontmatter' into patch/format-week3-transcripts

jgagne committed:
Update "Owltastic” URL

Update to “https” and without “www”.

jgagne committed:
Remove comments

Tidy up.

jgagne committed:
Update external links

Add “target” and “rel” attributes.

jgagne committed:
Update formatting

- Add <strong> to callout the Resources section
- Add <cite> for citing the title of the tutorial
- Add emphasis for action, tone and voice

jgagne committed:
Update formatting

- Add <strong> to callout the Resources section
- Add <strong> to callout style attributes
- Add emphasis for action, tone and voice

jgagne committed:
Update formatting

- Add <strong> to callout the Resources section
- Add <kbd> for keyboard commands
- Add <strong> for callouts and typed phrases
- Add emphasis for action, tone and voice
- Update link text

jgagne committed:
Update formatting

- Add <strong> to callout the Resources section
- Add emphasis for action, tone and voice

jgagne committed:
Update formatting

- Add <strong> to callout the Resources section
- Add <strong> to callout the CodePen theme
- Add <cite> for citing CodePen
- Breakup instruction steps into separate paragraphs
- Add emphasis for action, tone and voice

jgagne committed:
Update sign-off formatting

Update emphasis.
